bundle/internal/validation/generated/enum_fields.go

```diff
@@ -0,0 +1,131 @@
+package generated
+
+// THIS FILE IS AUTOGENERATED.
+// DO NOT EDIT THIS FILE DIRECTLY.
+
+import (
+	_ "github.com/databricks/cli/libs/dyn"
+)
+
+// EnumFields maps [dyn.Pattern] to valid enum values they should have.
+var EnumFields = map[string][]string{
+	"resources.apps.*.active_deployment.mode": {"AUTO_SYNC", "SNAPSHOT"},
+	"resources.apps.*.active_deployment.status.state": {"CANCELLED", "FAILED", "IN_PROGRESS", "SUCCEEDED"},
```

Comment:
Validation will only take place if a user explicitly specifies a field in their bundle YAML. So having these output-only fields leak into this generated file is fine.

Right now, we don't have a good way to filter out output-only fields just based on the Go structs.